site/docs/v1/tech/apis/identity-providers/_samlv2-response-body.adoc

```diff
@@ -24,6 +24,22 @@ Determines if a UserRegistration is created for the User automatically or not. I
 [field]#identityProvider.applicationConfiguration``[applicationId]``.enabled# [type]#[Boolean]#::
 Determines if this identity provider is enabled for the Application specified by the `applicationId` key.
 
+ifndef::samlv2_idp_initiated[]
+[field]#identityProvider.assertionConfiguration.destination.alternates# [type]#[Array]# [since]#Available since 1.43.0#::
```

Comment:
In the request description you add this:

```
... if the [field]#identityProvider.assertionConfiguration.destination.policy# setting is `AllowAlternates`.
```

Can we use this same description here? Or at a minimum, we should use the proper named value in the API doc which is `AllowAlternates`.